apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -7,17 +7,18 @@ import {
   Select,
```

Comment:
I think there should be some wrong configuration of the linter as .ts and .tsx files use different linting

Reply:
Seems to be the same (single quote and semi-colons), I just haven't applied it to some files

Reply:
Previously the projects used different configuration, now it's one configuration but I haven't formatted/linted the whole project so there can be some inconsistencies

Reply:
Yes, I'm fine to start linting modified files, but I was confused to see different rules applied to some files where the linter clearly run